tetris.py

```diff
@@ -104,6 +106,16 @@ def right_mediator(self, event):
         self.draw_block()
 
 
+color_chart = {
+    1: "turquoise",
+    2: "yellow green",
+    3: "yellow",
+    4: "gold",
+    5: "orange",
+    6: "dark orange",
+    7: "red2",
+    8: "red4",
+}
 class TetrisGame:
     def __init__(self):
         self.landed_blocks = []
```

Comment:
Change this to dict with coords as keys and colors as values:

```suggestion
        self.landed_blocks = {}
```

Of course, you need to change all code that uses `landed_blocks`.